docker-compose-microservices.yml

```diff
@@ -0,0 +1,108 @@
+version: '2'
+services:
+    artemis-app:
+        image: ghcr.io/ls1intum/artemis-service
```

Comment:
This configuration does not work at the moment. Docker-compose implicitly uses the `latest` tag if no other tag is specified. We don't have `latest` tagged images at the moment, as these are only created on the develop branch. 
As soon as this gets merged to develop, this file will work, but the behavior of it might not be anticipated by users of this file. 
It think we should introduce a env variable which specifies the image tag. Eg `${TAG}`. This tag can be specified using a `.env` file or as an CLI parameter. ( Ref: https://docs.docker.com/compose/environment-variables/) 

The image line could be updated to: 
```
image: ghcr.io/ls1intum/artemis-service:${TAG:-latest}
```
The default value is then explicitly defined as latest if no tag is specified.